ghstack/config.py

```diff
@@ -62,7 +59,15 @@ def read_config(*, request_circle_token: bool = False) -> Config:  # noqa: C901
     if config.has_option('ghstack', 'github_url'):
         github_url = config.get('ghstack', 'github_url')
     else:
-        github_url = "github.com"
+        github_url = input('GitHub url [github.com]: ')
```

Comment:
I'm not so keen on requesting the `github.com` URL explicitly from the user here, since the uber common case is that you're just using normal github.com (and you should just do a manual intervention if you're enterprise).

Reply:
So this is just one time, do you think this is an issue? Right now for enterprise I need to ask users to copy paste the config. I thought this would be a better experience. While for github.com case people just hit enter only one time and forget about it. If you think that this is not mergable, then I guess there is no point in this PR :)

Reply:
Alright fine. But will you fix the other issues then? :)

Reply:
yep, done :)